MarvellOS.py

```diff
@@ -221,6 +223,53 @@ def tick1(self):
         global truth
         if truth:
             welcome_screen.after_cancel(tick)
+    def gotocalendar(self):
+        self.gotowin=Tk()
+        self.gotowin.title("Year Calendar")
+        self.gotowin.lift()
+        self.gotowin.attributes('-topmost', True)
+        self.gotowin.geometry("550x600")
```

Comment:
We need to have the size of each window to be the same.
So, you can just use the default size, and see the idea behind the implementation in the review comment.